subprojects/dependency-management/src/main/java/org/gradle/api/internal/artifacts/result/DefaultResolvedComponentResult.java

```diff
@@ -104,17 +105,16 @@ public ModuleVersionIdentifier getModuleVersion() {
     }
 
     @Override
-    @SuppressWarnings("deprecation")
```

Comment:
I think this has to stay around since we have `-Werror` turned on in this project.